libs/api/transaction/data-access/src/lib/api-transaction-data-access.service.ts

```diff
@@ -163,6 +172,7 @@ export class ApiTransactionDataAccessService implements OnModuleInit {
     const appTransaction: AppTransactionWithErrors = await this.createAppTransaction({
       appEnvId: appEnv.id,
       commitment: input.commitment,
+      ip: req.ip,
```

Comment:
Same as the above...


Probably a better idea to store `const ip = req.clientIp` and reference that, instead of using `req.ip` in 3 places.